src/platform/site-wide/sass/modules/_m-homepage-blog.scss

```diff
@@ -0,0 +1,27 @@
+.homepage-blog {
+  &__image {
+    height: 100%;
+
+    img {
+      width: 100%;
+      height: 350px;
+      object-fit: cover;
+      object-position: top;
+      display: block;
+
+      @media (min-width: $small-screen) {
+        height: 100%;
+      }
+    }
+  }
+
+  .va-h-ruled--white {
+    border-bottom: 1px solid $color-white;
```

Comment:
Can we use design system classes instead of custom CSS here? https://design.va.gov/foundation/utilities/border

Reply:
@timwright12 The only divider in the design system contains star SVGs (https://design.va.gov/components/divider), and doesn't seem like any other variations of the divider component exist, unless they do exist and I'm not looking in the right place?

Reply:
I can remove this altogether and create a div with some utility classes. That way we don't have to define anything here.